skimmer/skimmer_utils.py

```diff
@@ -31,6 +31,22 @@ def update_cut_flow(cut_flow, cut_name, events):
         cut_flow[cut_name] = get_number_of_events(events)
 
 
+def add_variations_to_cutflow(cut_flow, var_name, nominal, up, down):
+    """Add normalization factors for nominal/up/down variations
+
+    Args:
+        cut_flow (dict[str, float])
+        var_name (str): the name of the cut to appear in the cut flow tree
+        nominal (float)
+        up (float)
+        down (float)
+    """
+
+    cut_flow[f"sumw_{var_name}_nominal"] = nominal
```

Comment:
It's the same, but not sure this is always the case

Reply:
Not sure why it would be different.
We can keep both for convenience later in the code, but I find it more confusing two have twice the same information than having it once if it's the same.

Reply:
If e.g. we were to apply a scale factor, this would move the central value and then have its up/down variations. It's true that in this case it is duplicate info, but since it's not always the case I'd argue that we can afford to store one extra float